### PR TITLE
Added Tile.GetGasPressure function

### PR DESCRIPTION
**Pull request by Dan54**
Added `Tile.GetGasPressure(string gas)`. If `Tile` is in a room it returns `Tile.Room.GetGasPressure(gas)` otherwise it returns the lowest pressure from `Tile`'s neighbors that are in a room.

### CODE REVIEW

**Comment by Randshot** [comment]
I have no problems running the code and the performance seems to be about the same.
One thing to note is that this PR has some (many) whitespace changes.
Nothing world changing but wanted to mention it anyway.

**Comment by Dan54** [comment]
I realized after making my changes that I had left tabs in the code, so I replaced all tabs with spaces.

**Comment by Randshot** [comment]
@Dan54 I don't mind but wanted to mention it for the Gatekeepers.
They will have to decide what to do :)

**Comment by Raketfart** [comment]
I've read this through, and it seems good. Haven't tested it though.

File: Assets/Scripts/Models/Tile.cs [hunk 1]
```diff
@@ -316,6 +316,40 @@ public Tile West()
...
+    public float GetGasPressure(string gas)
+    {
+        if (Room == null)
+        {
+            float pressure = Mathf.Infinity;
+            if (North().Room != null && North().GetGasPressure(gas) < pressure)
+            {
+                pressure = North().GetGasPressure (gas);
+            }
+            if (East().Room != null && East().GetGasPressure(gas) < pressure)
```
Comment (quill18) [inline Assets/Scripts/Models/Tile.cs]: I might be missing something, but shouldn't this be an `else if`?

Comment (Raketfart) [inline Assets/Scripts/Models/Tile.cs]: No, it takes the lowest gas pressure from all adjacent tiles

**Comment by Mizipzor** [comment]
This branch has conflicts that must be resolved, probably due to the huge StyleCop cleanup that just happened. Fix them and remove the `[Conflicts]` tag. :)

**Comment by Mizipzor** [comment]
Looks good. :) The gas and pressure systems are a bit in a state of flux at the moment, I'm going to be lenient with pull requests for it.

**Pull request merged**